exchange/client.go

```diff
@@ -12,10 +15,30 @@ func BuildHTTPClient(options *Options) (*http.Client, error) {
 	if options.FollowRedirects {
 		checkRedirect = nil
 	}
+	tlsConf := &tls.Config{}
+
+	if options.SkipVerify {
+		tlsConf.InsecureSkipVerify = true
+	}
+
+	transp := &http.Transport{
+		Proxy:           http.ProxyFromEnvironment,
+		TLSClientConfig: tlsConf,
+		// ForceAttemptHTTP2: true,
+	}
+
+	if !options.ForceHTTP1 {
+		err := http2.ConfigureTransport(transp)
```

Comment:
I think HTTP2 is enabled by default.
So this line does not required.

Reply:
Done. I had to do that with de original http transport, once cloned from the default, it's no more needed.